public/stylesheets/main.css

```diff
@@ -37,3 +38,77 @@ h3 {
 }
 
 /* TODO - Add your own styles here */
+
+/* Headings */
+
+#page-title {
+font-style: italic;
+font-size: 5em;
+text-align: center;
+}
+
+div h2 {
+  text-align: center;
+  font-size: 3em;
+  font-weight: bold;
+  margin-bottom: 10px;
```

Comment:
Be consistent in your CSS - if you're using ems for fonts might as well make your margins ems too...